docs/content/docs/dev/table/sql/queries/window-tvf.md

```diff
@@ -289,6 +289,100 @@ Here is an example invocation on the Bid table:
 +------------------+------------------+-------+
 ```
 
+### SESSION
+
+{{< hint info >}}
+Note:
+1. Session Window TVF is not supported in Batch mode now.
+2. Session Window Aggregation is not supported [Performance Tuning]({{< ref "docs/dev/table/tuning" >}}) now.
+3. Session Window Join, Session Window TopN and Session Window Deduplication are in experimental phase now.
```

Comment:
What do you mean by experimental?

Reply:
The `Session Window join`, `Session Window TopN` and `Session Window Deduplication` is not fully verified about correctness and etc. You can find more here: https://issues.apache.org/jira/browse/FLINK-34315. However, the pr in that Jira may not be merged because session window tvf is not supported in previous version, and will not affect the correctness about legacy plan. What we only need to do is to add `Note` in doc about this part.